style.css

```diff
@@ -277,6 +290,85 @@ body {
   font-weight: 500;
 }
 
+/* testimonial-section */
+
+.second-content-section {
+  background-color: #fafafa;
+}
+
+.section-testimonials {
+  padding: 12rem 0;
+}
+
+.testimonils {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  column-gap: 1.5rem;
+}
+
+.testimonial-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  background-color: #fff;
+  color: #737373;
+  padding: 3rem 3.5rem;
+  border: 1px solid #dedede;
+  border-radius: 5px;
+  margin: 0 1.5rem;
+  gap: 2rem;
+}
+
+.testimonial-text {
+  font-size: 1.4rem;
+  font-weight: 600;
+  line-height: 1.5;
```

Comment:
This seems to be a very common value "  line-height: 1.5;" among your CSS file, in that case, you put it into a CSS Variable, this will ensure consistency among the whole design.

That could be applied to theme colours, font size and other fixed values among the design.

reference: https://www.w3schools.com/css/css3_variables.asp

Reply:
yup that's my plan for cleanup, I'm gonna make variables for all common and frequent use values such as font-size font-weight, etc,